src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -218,7 +218,7 @@ export const terminalConfiguration: IConfigurationNode = {
 			default: false
 		},
 		'terminal.integrated.commandsToSkipShell': {
-			markdownDescription: localize('terminal.integrated.commandsToSkipShell', "A set of command IDs whose keybindings will not be sent to the shell and instead always be handled by VS Code. This allows the use of keybindings that would normally be consumed by the shell to act the same as when the terminal is not focused, for example ctrl+p to launch Quick Open. Use the command prefixed with `-` to remove default commands from the list.\nDefault Skipped Commands:\n\n{0}", DEFAULT_COMMANDS_TO_SKIP_SHELL.sort().map(command => `- ${command}`).join('\n')),
+			markdownDescription: localize('terminal.integrated.commandsToSkipShell', "A set of command IDs whose keybindings will not be sent to the shell but instead always be handled by VS Code. This allows keybindings that would normally be consumed by the shell to act instead the same as when the terminal is not focused, for example `Ctrl+P` to launch Quick Open.\n\n---\n\nMany commands are skipped by default. To override a default and pass that command's keybinding to the shell instead, add the command prefixed with the `-` character. For example add `-workbench.action.quickOpen` to allow `Ctrl+P` to reach the shell.\n\n*The following list of default skipped commands is truncated when viewed in Settings Editor. To see the full list, click [here](command:workbench.action.keepEditor 'Keep this editor open'), then [here](command:workbench.action.openRawDefaultSettings 'Open Default Settings (JSON)'), then search for the first command from the list below.*\n\n---\n\nDefault Skipped Commands:\n\n{0}", DEFAULT_COMMANDS_TO_SKIP_SHELL.sort().map(command => `- ${command}`).join('\n')),
```

Comment:
> To see the full list, click [here](command:workbench.action.keepEditor 'Keep this editor open'), then [here](command:workbench.action.openRawDefaultSettings 'Open Default Settings (JSON)'), then search for the first command from the list below.

Using [here] is bad practice for screen readers, I suggest:

> To see the full list, [open the default settings JSON](command:workbench.action.openRawDefaultSettings 'Open Default Settings (JSON)') and search for the first command from the list below.

I removed the first part as 2 steps complicates things a bit.

Reply:
@Tyriar thanks for the feedback. I have pushed a change.